scripts/shared/e2e.sh

```diff
@@ -61,7 +60,7 @@ function test_with_e2e_tests {
 
 ### Main ###
 
-declare_kubeconfig
 [[ "${lazy_deploy}" = "false" ]] || deploy_env_once
+declare_kubeconfig
```

Comment:
I suspect this change could break consuming projects, since if you dont declare the kubeconfig before checking the lazy deploy thing, it wont ever find an env even if it is deployed.
Have you checked if other projects still behave as expected with this change present?

Reply:
I wasn't aware of this.  I moved it b/c it yields an error "_find: '/go/src/github.com/submariner-io/shipyard/output/kubeconfigs': No such file or directory_" if no clusters exist prior to the check. But it'd benign - I'll undo this change to be safe.

Reply:
Yeah if it wasnt deployed you'll see this printed, we can fix that separately though